JobConfig/primary/CeEndpoint.fcl

```diff
@@ -8,7 +8,7 @@ physics.producers.generate : {
   module_type : CeEndpoint
   inputSimParticles: TargetStopResampler
   stoppingTargetMaterial : "Al"
+  pdgId : 11
   verbosity : 0
 }
-physics.producers.FindMCPrimary.PrimaryProcess : "mu2eCeMinusEndpoint"
```

Comment:
Why this line was removed? Maybe the tests are failing because of this?

Reply:
oops, meant to remove it from another file because of duplication. I will add back here and remove the duplicate in the "steps" version